### PR TITLE
Adding image fresnel zone

### PR DESCRIPTION
**Pull request by tidelbo**

### CODE REVIEW

File: docs/2/installer/lorawan/gateway-installations/outdoor-istation.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ title: Kerlink iStation
 description: How to set it up?
 toc: true
 ---
+## Planning your install
+
+{% include image.html width=400 url="/assets/img/LoRaWAN Device pictures/Without text.jpg" description="LoRaWan Technology" %}
```
Comment (JulienBardin) [inline docs/2/installer/lorawan/gateway-installations/outdoor-istation.md:9]: no space, it should be dashes and lower case. Take a look at the other images in the same page

File: docs/2/installer/lorawan/gateway-installations/outdoor-istation.md [hunk 2]
```diff
@@ -125,4 +132,7 @@ Once all secure, turn on the power of the PoE injector. You can now start deploy
 
 You’re all done! Please send an email to support@microshare.io to indicate the Gateway is up and running so we can confirm at our end.
 
- 
\ No newline at end of file
+---------------------------------------
+
+### Reminder: for optimal performances
+{% include image.html width=400 url="/assets/img/LoRaWAN Device pictures/Without text.jpg" description="LoRaWan Technology" %}
```
Comment (JulienBardin) [inline docs/2/installer/lorawan/gateway-installations/outdoor-istation.md:138]: no space, it should be dashes and lower case. Take a look at the other images in the same page

**Review by JulienBardin — CHANGES_REQUESTED** [review]
@tidelbo some edits are need otherwise good to go

**Comment by JulienBardin** [comment]
you use two time the same image and you don't use : assets/img/LoRaWAN Device pictures/Without text.jpg

**Comment by tidelbo** [comment]
> you use two time the same image and you don't use : assets/img/LoRaWAN Device pictures/Without text.jpg

Yes is is two time the same image on purpose. And I changed the image that is why it is not the same name anymore

**Comment by JulienBardin** [comment]
Can you delete the image not used ?

**Comment by tidelbo** [comment]
> Can you delete the image not used ?

It is done.

**Pull request merged**